src/components/Announcements.js

```diff
@@ -8,7 +9,9 @@ class Announcements extends React.Component {
   }
 
   async componentDidMount() {
-    const response = await fetch("http://localhost:3001/announcements");
+    const response = await fetch("http://localhost:3001/announcements", {
+      headers: { authorization: localStorage.get("token") }
+    });
```

Comment:
you should check if response is 401 error and redirect if so

Reply:
we need to check response.status and redirect if 401 before calling response.json. otherwise response.json throws an error. To check this you can clear your localstorage and then try to access /orders URL and see if you are redirected. Needs to be updated announcements, orders, and inventory.